src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -169,7 +169,7 @@ var HTMLDOMPropertyConfig = {
     // Setting .type throws on non-<input> tags
     type: MUST_USE_ATTRIBUTE,
     useMap: null,
-    value: MUST_USE_PROPERTY | HAS_SIDE_EFFECTS,
+    value: MUST_USE_ATTRIBUTE,
```

Comment:
If `HAS_SIDE_EFFECTS` is now unused then the corresponding logic should be removed from `DOMPropertyOperations` as well? (PS. Also, this is outdated and needs to be rebased)